docker-compose-services/postgres/README.md

```diff
@@ -38,6 +38,10 @@ Two new `ddev` commands are provided:
 Example `config.yaml` hooks configuration to automatically import/export the `db` table:
 
 ```
+# Add psql to your webserver
+webimage_extra_packages: [postgresql-11]
```

Comment:
This should be postgresql-client, not postgresql-11, which would install the whole package, including server.

Reply:
Right, less is more 👍